config/types/bounty.ts

```diff
@@ -50,6 +51,7 @@ export default class Bounty {
     this._timeRemainingInDays = timeToInUnits({ date: raw.expiration_date, unit: "day" });
     this._createdBy = parseCreatedBy(raw.created_by);
     this._amount = parseFloat(raw.amount);
+    this._formattedAmount = parseInt(this._amount.toFixed(0)).toLocaleString();
```

Comment:
I don't like parsing int like this -- what if the actual number is something that isn't an int? I think we should change

Reply:
One big thing to consider is that RSC has 18 decimals. So whenever we work with RSC we should always allow for the possibility of non integers (but our inputs can force integers, just when we read things back they should always allow for non integers)

Reply:
Yeah makes sense.